### PR TITLE
Update NetNamedPipeBinding.xml

### PR DESCRIPTION
**Pull request by chrysikos**
Important info about port concept.

### CODE REVIEW

**Comment by rpetrusha** [comment]
@davidsh, could you take a look, please?

**Comment by rpetrusha** [comment]
@mconnew and @Lxiamail, could you review this please?

**Comment by mconnew** [comment]
Besides the grammar issues, I don't believe this change needs to be made for several reasons.  
1. NetNamedPipeBinding doesn't accept a URI on any api's so talking about the supported URI's here seems out of place.
2. NetNamedPipeBinding isn't a transport, it's a standard binding which combines binding elements together in any easy way. NamedPipeTransportBindingElement is the actual transport binding element which cares about the URI. Even that class doesn't have any api's which use a URI, that would be the ChannelFactory created by NamedPipeTransportBindingElement.  
3. Documenting this is setting a precedent. Would we need to document NetMsmqBinding, MsmqIntegrationBinding and NetPeerTcpBinding?
4. The port number of the URI is just one of many differences about the URI between different transports. For example, using a URL fragment isn't applicable to anything other than HTTP. Documenting core concepts such as the HTTP protocol or what is a named pipe is out of scope for WCF documentation. 
5. A port number is only applicable when using a TCP connection (which HTTP and Net.Tcp both do). It's clearly documented that a named pipe is used, which is not using TCP/IP so a port number is not applicable. The documentation on the same page says (emphasis mine):
> The NetNamedPipeBinding generates a run-time communication stack by default, which uses transport security, **named pipes** for message delivery

Basically it's clearly documented that this uses named pipes, and it's expected that if someone is choosing to use a named pipe for their transport, that they have a basic understanding of what a named pipe is before using it. Without that basic understanding, you can't make an educated decision about if the transport is the right one to use.

**Comment by rpetrusha** [comment]
Thank you for contributing to the dotnet/docs repo, @chrysikos. However, based on the review from @mconnew, I'm afraid that we can't accept your PR, so I'm going to close it.
